tests/testthat/test-full-seq.R

```diff
@@ -1,31 +1,36 @@
 test_that("works with numeric", {
   x <- c(0, 100)
-  expect_equal(fullseq(x, 50), c(0, 50, 100))
+  expected <- c(0, 50, 100)
+  expect_equal(fullseq(x, 50), expected)
+  expect_equal(fullseq(rev(x), 50), expected)
 })
 
 test_that("works with POSIXct", {
   x <- as.POSIXct(c("2000-01-01 08:29:58", "2000-01-01 08:30:10"), tz = "UTC")
 
-  expect_equal(
-    fullseq(x, "1 hour"),
-    as.POSIXct(c("2000-01-01 8:00:00 UTC", "2000-01-01 9:00:00 UTC"), tz = "UTC")
-  )
+  expected <- as.POSIXct(c("2000-01-01 8:00:00 UTC", "2000-01-01 9:00:00 UTC"), tz = "UTC")
+  expect_equal(fullseq(x, "1 hour"), expected)
+  expect_equal(fullseq(rev(x), "1 hour"), expected)
+
   expect_equal(
     fullseq(x, ".5 secs")[1:2],
     as.POSIXct(c("2000-01-01 08:29:58.0 UTC", "2000-01-01 08:29:58.5 UTC"), tz = "UTC")
   )
 })
 
 test_that("works with Date", {
-  x <- as.Date("2012-01-01") + 1:30
-  expect_equal(fullseq(x, "1 month"), as.Date(c("2012-01-01", "2012-02-01")))
```

Comment:
This test was wonky. `fullseq()` only considers `range[1]` and `range[2]`, but this is a vector of 30 dates instead of a range of 2 dates.

In addition, `ceiling_date("2012-01-01", "1 month")` gives 1st of march because 31 days from the end of January is in March due to February typically having only 28 days. I consider this out of scope for this PR, but it isn't doing any real harm anyway.